.github/workflows/test.yml

```diff
@@ -0,0 +1,34 @@
+name: Test
+
+on:
+  workflow_call:
+    inputs:
+      node_versions:
```

Comment:
Would it be possible to use a comma-separated string (or something like that) here if arrays aren't supported? I guess the challenge will be processing that string - if it was used in a step, we could run a step beforehand to transform the value, but `strategy.matrix` will be evaluated before the steps run, I would imagine.

I wonder if you might be able to use dependent jobs to process the `node_versions` setting and make it available to the next step, like https://www.macstadium.com/blog/sharing-variables-between-jobs-in-github-actions.

Reply:
we can use the [`fromJson` helper](https://docs.github.com/en/actions/learn-github-actions/expressions#fromjson), I'm curious to see if this will work

```yml
  with:
    node_versions: "[14,16,18]"
```

and then

```yml
  test_matrix:
    runs-on: ubuntu-latest
    strategy:
      matrix:
        node_version: ${{ fromJson(inputs.node_versions) }}
```

Reply:
I didnt know about this helper! It looks promising and would be a lot slicker than dependent jobs.

Reply:
`fromJSON()` works 🥳 👍🏽 
https://github.com/octokit/auth-oauth-user.js/pull/71

Reply:
Awesome, thanks for verifying!